packages/odyssey-storybook/src/components/odyssey-mui/CssBaseline/CssBaseline.stories.tsx

```diff
@@ -593,7 +593,7 @@ export const Sup: StoryObj = {
 };
 
 export const Ul: StoryObj = {
-  render: function C() {
+  render: function C({}) {
```

Comment:
What caused this change?

Reply:
That made the `show code` did not return `render function`. That case, it didn't copy the unwanted code when copy code and also look nicer.

Before the change
```
{
  render: function C() {
    return <ScopedCssBaseline>
        <p>
          Remember that <abbr title="Faster than Light">FTL</abbr> travel may
          cause subjective time distortions.
        </p>
      </ScopedCssBaseline>;
  }
}
```

After
```
<ScopedCssBaseline2>
  <p>
    Remember that{' '}
    <abbr title="Faster than Light">
      FTL
    </abbr>
    {' '}travel may cause subjective time distortions.
  </p>
</ScopedCssBaseline2>
```